jina/types/sets/document.py

```diff
@@ -219,7 +219,7 @@ def stack_embeddings(embeddings):
             elif sparse_cls_type == 'tf':
                 import tensorflow as tf
 
-                return tf.sparse.concat(axis=0, sp_inputs=embeddings)
+                return embeddings
```

Comment:
how does this work? Is there still a need to import tf?

Reply:
oh good point